app/controllers/api/scores_controller.rb

```diff
@@ -15,6 +15,20 @@ def user_feed
       render json: response.to_json
     end
 
+    def golfer
+      link_id = params[:golfer_id]
+      golfer_scores = Score.where(user_id: link_id)
+      golfer_name = User.find_by(id: link_id).name
+      serialized_scores_golfer = golfer_scores.map(&:serialize)
+      response = {
+        golfer: {
+          golfer_name: golfer_name,
+          golfer_scores: serialized_scores_golfer,
+        }
+      }
+      render json: response.to_json
```

Comment:
Let's separate the logic in this function with empty lines as follows:

- Variable declaration
- Response prepare
- Response return

Reply:
I think you don't need `link_id`. You can use params[:golfer_id] directly